ballerina/modules/serializer/tests/schema_test.bal

```diff
@@ -72,7 +72,8 @@ function constructShape(json data) returns json|schema:SchemaError {
 
 @test:Config {
     dataProvider: customTagDataGen,
-    groups: ["serializer"]
+    groups: ["serializer"],
+    enable: false
```

Comment:
Why did we disable this?

Reply:
This is due to a bug in the implementation. Will enable it once it is fixed. Created issue: https://github.com/ballerina-platform/ballerina-standard-library/issues/4480

Reply:
Implementation wasn't changed when this was made a standard library[[1]](https://github.com/ballerina-platform/ballerina-standard-library/issues/4288). This is a pre-release so this will be addressed in the next release.

Reply:
Is this failing only when we change the `ballerina/regex` to `lang.regexp`?

Reply:
Enabled in [3ec2028](https://github.com/ballerina-platform/module-ballerina-yaml/pull/9/commits/3ec20285a3af4c4280566410f3b825814c2d6c78)